index.js

```diff
@@ -0,0 +1,24 @@
+const menu = document.querySelector('.menu');
+
+const manipulateMenu = () => {
+  menu.classList.remove('d-hidden');
+};
+
+const menuListener = () => {
+  const menuIcon = document.querySelector('.fa-bars');
+  menuIcon.addEventListener('click', () => {
+    manipulateMenu();
+  });
+};
+
+document.addEventListener('DOMContentLoaded', () => {
+  menuListener();
+});
+
+menu.addEventListener('click', (event) => {
+  const name = event.target.localName;
+  console.log(name);
```

Comment:
### OPTIONAL

- [ ] You should not use console.log for best practice.

You can check it in curriculum. [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20console.log%20to%20your%20repo.%20It%E2%80%99s%20ugly%2C%20it%20kills%20performance%20and%20it%20can%20make%20confidential%20data%20be%20visible%20to%20anyone%20using%20the%20browser%20tools%20to%20look%20at%20your%20website.)